pkg/diskmaker/controllers/lvset/reconcile.go

```diff
@@ -165,6 +178,26 @@ DeviceLoop:
 				continue DeviceLoop
 			}
 		}
+
+		getTime := *r.deviceAgeMap.getTime
```

Comment:
I am not sure about - a function that just returns `time.Now()` leaking through here.